src/seals/util.py

```diff
@@ -245,26 +266,27 @@ def step(self, action):
             obs = self.absorb_obs_this_episode
             rew = self.absorb_reward
             info = {}
+            truncated = False
 
-        return obs, rew, False, info
+        return obs, rew, False, truncated, info
 
 
 def make_env_no_wrappers(env_name: str, **kwargs) -> gym.Env:
-    """Gym sometimes wraps envs in TimeLimit before returning from gym.make().
+    """Gym sometimes wraps envs in TimeLimit before returning from gymnasium.make().
 
     This helper method builds directly from spec to avoid this wrapper.
     """
-    return gym.envs.registry.env_specs[env_name].make(**kwargs)
+    return gym.spec(env_name).make(**kwargs)
```

Comment:
`spec.make` might get deprecated soon-ish, plus it's just an alias for `gym.make(self)`, so I don't think this will work?

`TimeLimit` is applied in two cases:
- `EnvSpec` has a `max_episode_steps` applied
- `max_episode_steps` is directly given in `gym.make`

As I understand, the intention here is that no matter what the env is, we remove the time limit?

I can see the following options to handle this:
- Copy the spec, remove the time limit, create from that
- Use `env.unwrapped` to strip all the default wrappers
- Recursively go up the wrapper hierarchy and rip out the `TimeLimit`

It's worth noting that since recently, `EnvSpec` can specify the list of wrappers that are applied by default, so there might be some subtle interactions to keep in mind here

Reply:
I removed it since it was only used in a single point.